src/lib/viewers/controls/annotations/AnnotationsControls.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import noop from 'lodash/noop';
+import AnnotationsButton from './AnnotationsButton';
+import IconHighlightText16 from '../icons/IconHighlightText16';
+import IconRegion24 from '../icons/IconRegion24';
+import useFullscreen from '../hooks/useFullscreen';
+import { AnnotationMode } from './types';
+import './AnnotationsControls.scss';
+
+export type Props = {
+    annotationMode?: AnnotationMode;
+    fileId: string;
+    onAnnotationModeClick?: ({ mode }: { mode: AnnotationMode }) => void;
+    hasHighlight?: boolean;
+    hasRegion?: boolean;
+};
+
+export default function AnnotationsControls({
+    annotationMode = AnnotationMode.NONE,
+    fileId,
+    onAnnotationModeClick = noop,
+    hasHighlight = true,
+    hasRegion = true,
+}: Props): JSX.Element | null {
+    const isFullscreen = useFullscreen();
+    const showHighlight = !isFullscreen && hasHighlight;
+    const showRegion = !isFullscreen && hasRegion;
+
+    if (!showHighlight && !showRegion) {
+        return null;
+    }
+
+    const handleModeClick = (mode: AnnotationMode): void => {
+        onAnnotationModeClick({ mode: annotationMode === mode ? AnnotationMode.NONE : mode });
+    };
+
+    return (
+        <div className="bp-AnnotationsControls">
+            <AnnotationsButton
```

Comment:
Is the `isEnabled` pattern preferred over doing a conditional render like `{ showRegion && <AnnotationsButton .../>`

Reply:
I don't think there is a strong signal either way. If it's just a preference, then it seems okay as-is?